Targets/STM32F4/STM32F4_CAN.cpp

```diff
@@ -0,0 +1,1597 @@
+// Copyright Microsoft Corporation
```

Comment:
If this file was not originally copyright Microsoft or Oberon, don't include them here.